avocado/plugins/runner_nrunner.py

```diff
@@ -267,6 +267,16 @@ def run_suite(self, job, test_suite):
             job.interrupted_reason = f"Suite {test_suite.name} is disabled."
             return summary
 
+        spawner_name = test_suite.config.get("run.spawner")
+        spawner = SpawnerDispatcher(test_suite.config, job)[spawner_name].obj
+        if not spawner.is_operational():
+            suite_name = f" {test_suite.name}" if test_suite.name else ""
+            msg = f'Spawner "{spawner_name}" is not operational, aborting execution of suite{suite_name}. Please check the logs for more information.'
```

Comment:
I guess one would naturally look into the logs next which would allow us to not have to provide a long message like this comment can be fully ignored.